spec/std/colorize_spec.cr

```diff
@@ -1,7 +1,7 @@
 require "spec"
 require "colorize"
 
-private def colorize(obj, *args)
+private def colorize_wrap(obj, *args)
```

Comment:
why?

Reply:
`Scope.colorize` is resolved in `it` blocks before file-scope `colorize` method, so it needs to change name.